sorting/merge_sort/multithreaded_merge_sort_golang/go.mod

```diff
@@ -0,0 +1,3 @@
+module multithreaded_mergesort
+
+go 1.23.3
```

Comment:
is this necessary?
instead rename file name to `multi_threaded_merge_sort.go`

Reply:
So you only want the code file and not the Go project setup ?

Reply:
yes

Reply:
I have removed the project setup and renamed the file